### PR TITLE
Alusine - JavaScript3 homework week1

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -30,13 +30,93 @@
...
+        class: 'contributor-details',
+      });
+      createAndAppend('img', contributorsDetailsDiv, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'image',
+      });
+      createAndAppend('p', contributorsDetailsDiv, {
+        text: contributor.login,
+        href: contributor.html_url,
```
Comment (remarcmij) [inline homework/index.js]: An `href`` attribute is not allowed on a `p` element. This is the full error report from the HTML Validator on the HTML your app generates (the line number refers to the generated HTML and have no relation to this JavaScript file).

```
Error: Attribute “href” not allowed on element “p” at this point.
    From line 16, column 2454; to line 16, column 2530
    Error: Attribute “target” not allowed on element “p” at this point. 
    From line 16, column 2454; to line 16, column 2530
    Error: Attribute “href” not allowed on element “p” at this point.   
    From line 16, column 2709; to line 16, column 2788
    Error: Attribute “target” not allowed on element “p” at this point. 
    From line 16, column 2709; to line 16, column 2788
```

File: homework/index.js [hunk 2]
```diff
@@ -30,13 +30,93 @@
...
+      }
+    });
+  }
+
+  function dropDown(root, repos) {
+    const header = createAndAppend('div', root, { id: 'header' });
+    createAndAppend('p', header, { text: 'HYF Repositories', class: 'header' });
+    const select = createAndAppend('select', header, { id: 'select' });
+
+    repos.forEach((repo, index) => {
```
Comment (remarcmij) [inline homework/index.js]: The assignment asked for the repos to be sorted by repo name.

File: homework/style.css [hunk 3]
```diff
@@ -1,3 +1,81 @@
 .alert-error {
```
Comment (remarcmij) [inline homework/style.css]: Your app is not responsive:

image

File: homework/index.js (hunk 1 above)
Comment (alquinex) [reply homework/index.js]: Hi @jim, 
Thanks for the feedback. I have been so engaged with something else but will keep you updated once the changes are made. Thanks!

Comment (alquinex) [reply homework/index.js]: Hi @jim,
The changes are made except for the responsiveness which will be done in week2.

File: homework/index.js [hunk 4]
```diff
@@ -30,13 +30,95 @@
...
+      });
+    });
+  }
+
+  function fetchAndRenderData(selectedRepo, repoContainer, contributorsContainer, root) {
+    repoContainer.innerHTML = '';
+    contributorsContainer.innerHTML = '';
+
+    const updatedAt = new Date(selectedRepo.updated_at);
+    createAndAppend('p', repoContainer, {
```
Comment (remarcmij) [inline homework/index.js]: The assignment asks that the repo name is a link that opens the current GitHub repo in a new browser tab.

**Pull request closed without merging**